### ISSUE TITLE
Remove deprecated czConfig

### ISSUE THREAD

**Issue by jimthedev** [commitizen/cz-cli#38]
#30 added deprecation warnings and switched defaults over to using config.commitizen in package.json. We just need to remove the deprecated code for 3.0.0 but this'll be a breaking change.

**Comment by jimthedev** [comment]
Closed by 3a3d10d52060b339b3108b5da61f90ba10b987a2, right @LinusU?

**Comment by LinusU** [comment]
Yes, I didn't know it I should close it now or when we merge v3.x into master :) up to you 👌
